internal/jujuapi/migrationtarget.go

```diff
@@ -30,13 +30,31 @@ func (api *API) Prechecks(model params.MigrationModelInfo) error
 func init() {
 	facadeInit["MigrationTarget"] = func(r *controllerRoot) []int {
 		preChecks := rpc.Method(r.Prechecks)
+		caCert := rpc.Method(r.CACert)
 
 		r.AddMethod("MigrationTarget", 4, "Prechecks", preChecks)
+		r.AddMethod("MigrationTarget", 4, "CACert", caCert)
 
 		return []int{4}
 	}
 }
 
+// CACert implements the CACert method of the MigrationTarget facade.
+// It is used by the source Juju controller to retrieve the CA cert of
+// the target controller during model migration, if the client did not
```

Comment:
We've gone into good detail here but it'd be good to elaborate why it might not send a CA cert and at which point? Otherwise it leaves the reader confused as to why this might happen.

Reply:
Changed